tools/CreateJsonSchema/Program.cs

```diff
@@ -0,0 +1,79 @@
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using Microsoft.Docs.Build;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Schema;
+using Newtonsoft.Json.Schema.Generation;
+using Newtonsoft.Json.Serialization;
+
+class Program
+{
+    static int Main(string[] args)
+    {
+        var license = Environment.GetEnvironmentVariable("JSON_SCHEMA_LICENSE");
```

Comment:
>license [](start = 12, length = 7)

So we have set the license on CI machine already? Otherwise, the limit is 10 generation per hour, which we have not reached yet.

Reply:
not yet, I don't have the keys :(